src/test/scala/cornichonTests/authTest.scala

```diff
@@ -5,9 +5,11 @@ class AuthTest extends CornichonFeature with FeatureConfig {
   def feature = Feature("checking the API request"){
     Scenario("API request returns 200"){
 
-      WithBasicAuth (configuration.clientId,configuration.clientSecret){
-        When I post(s"$authUrl")
-        .withParams("grant_type" -> "client_credentials", "scope" -> s"manage_project:$projectKey")
+      WithBasicAuth (clientId,clientSecret){
+        When I post(s"${authUrl}")
```

Comment:
No need to create a string to interpolates only one string. 😉 
```suggestion
        When I post(authUrl)
```

Reply:
done